package.json

```diff
@@ -15,7 +15,7 @@
     "watch": "encore dev --watch",
     "lint": "run-p lint:*",
     "lint:eslint": "eslint assets/js",
-    "lint:prettier": "prettier --check .github assets drush webpack.config.js babel.config.js .eslintrc.js postcss.config.js prettier.config.js docker-compose.yaml",
+    "lint:prettier": "prettier --check .github assets drush webpack.config.js babel.config.js .eslintrc.js postcss.config.js prettier.config.js .docker/docker-compose.yaml",
```

Comment:
you should remove docker-compose and instead add .docker entire folder (ordered by name)